src/nevermined/Assets.ts

```diff
@@ -66,6 +81,20 @@ export enum RoyaltyKind {
     Legacy
 }
 
+export enum PublishMetadata {
+    JustMarketplaceAPI,
```

Comment:
maybe change to `OnlyMetadataAPI` to be consistent with the names used in DIDResolvePolicy

Reply:
or better. This should not be an option at all. we always need to upload to the marketplace api not to break other functionalities like query. Only the others are optional

Reply:
:+1: